### PR TITLE
RDM-4832 Check Your Answers functional tests

### PR DESCRIPTION
**Pull request by rafalkalita**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-4832

### Change description ###
Tests for show and hide on the Check Your Answers page.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: test/features/showandhide/basicFields_CYA.feature [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  Scenario: Check Your Answers shows fields that were shown on a wizard page
+    When I do meet the condition for showing the field
+    Then the following fields will be visible on CYA page:
+      | Text Field      |
+      | Text Field 2    |
+      | Marital Status  |
+      | Text Field 3    |
+
+  Scenario: Check Your Answers hides fields from a hidden wizard page
```
Comment (andrewfolga) [inline test/features/showandhide/basicFields_CYA.feature]: How do we set the hidden page conditions here? It is not immedialtely visible from the background step. Maybe I cannot see it 🤔

File: test/stepDefinitions/validationSteps.js [hunk 2]
```diff
@@ -18,7 +18,8 @@ defineSupportCode(function ({ Given, When, Then, Before, After }) {
     console.log(dataTable);
   });
 
-    Given(/^the definition sheet '(.*)' looks like this$/, async function (sheetName, dataTable) {
+  // TODO: remove when RDM-5022 is merged
```
Comment (andrewfolga) [inline test/stepDefinitions/validationSteps.js]: Why TODO here?

File: test/features/showandhide/basicFields_CYA.feature (hunk 1 above)
Comment (rafalkalita) [reply test/features/showandhide/basicFields_CYA.feature]: Fair point. Added page showCondition

File: test/stepDefinitions/validationSteps.js (hunk 2 above)
Comment (rafalkalita) [reply test/stepDefinitions/validationSteps.js]: Will remove.

**Pull request merged**